NServiceBus.FileBasedRouting/routing.xsd

```diff
@@ -0,0 +1,32 @@
+﻿<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <xs:element name="endpoints">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element minOccurs="0" maxOccurs="unbounded" name="endpoint">
+          <xs:complexType>
+            <xs:sequence>
+              <xs:element minOccurs="1" maxOccurs="1" name="handles">
```

Comment:
My reasoning was why would one add the empty file. On the other hand, why not? I'll remove it.